keyboards/mini_elixivy/mini_elixivy.h

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2021 minibois
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+// Easier reading of layouts
+#define ___ KC_NO
+
+/* ANSI
+ * ,-------------------------------------------------------------.
+ * |00 |01 |02 |03 |04 |05 |06 |07 |08 |09 |0a |0b |0c |   0d|0e |
+ * |-------------------------------------------------------------|
+ * |10  |11 |12 |13 |14 |15 |16 |17 |18 |19 |1a |1b |1c |  1d|1e |
+ * |-------------------------------------------------------------|
+ * |20   |21 |22 |23 |24 |25 |26 |27 |28 |29 |2a |2b |2c    ||2e |
+ * |-------------------------------------------------------------|
+ * |30     ||32 |33 |34 |35 |36 |37 |38 |39 |3a |3b      |3b |3c |
+ * |-------------------------------------------------------------|
+ * |40  |41  |42  |           46         |49 |4a |4b |4c |4d |4e |
+ * `-------------------------------------------------------------'
+*/
+#define ansi( \
```

Comment:
```suggestion
#define LAYOUT_ansi( \
```

Reply:
Hey @zvecr ,
Yesterday when I made this PR one of the QMK Discord mods told me (through Discord) to change my layout names from "layout_ANSI" to "ansi" (and so on for the ISO layout), telling me "we don't accept uppercase".

I just want to be sure "LAYOUT_ansi" (and so on for the ISO layout) is the correct naming, as opposed to what I have in the PR now.

Just asking for clarification, on if upper-case is not accepted at all in the layout names, or if "LAYOUT_ansi" is the convention for QMK

Cheers,
minibois.